handbook/communication/index.md

```diff
@@ -122,3 +122,28 @@ Most meetings at Sourcegraph are video calls. We prefer [Zoom](https://zoom.us)
 
 - [Conflict resolution](./code_of_conduct.md#conflict-resolution)
 - [Clean escalation](./code_of_conduct.md#clean-escalation)
+
+## Brain Food
+
+Brain Food meetings are our internal knowledge sharing sessions. These meetings can be formal presentations or informal discussions and are open to anyone in the company. Subjects of talks are completely free and are not limited to tech discussions. Here are a few examples:
+
+- sharing internal team knowledge to the rest of the company
+- presenting a useful tool
+- presenting a personal project
+- hands on session / workshops
+- brainstorming session about an experimental subject
+- preparing an external talk
+
+Brain Food sessions take place every other friday and, to accomodate timezone differences, are split in two separate sessions:
+
+- EMEA friendly session time: 16:00 CET
+- NA friendly session time: 2:00PM PST
+
+Each session is 1-hour long, however the session is only as long as the group needs to present and wants to keep the discussion going. They will both be recorded and published on Slack. Speakers can sign-up for either one of them depending on their own timezone.
+
+Presentations can use one of two formats:
+
+- Lightning Talks: Around 5 to 7 minutes long
+- Presentation: Around 10 to 14 minutes long
+
+To sign-up, use this [Google Sheet](https://docs.google.com/spreadsheets/d/1HLVFH9JUxchPNH4FoIQAThorbpdvmTl0ndmZzmpjsuU/edit?usp=sharing). The talk will be added to the Google Calendar invitation which will be sent to everyone.
```

Comment:
who will be the person managing the calendar events?

Reply:
At the beginning I was thinking it could be done manually by me or @aidaeology ,then figure out a better way to register and create the calendar invite automatically.

Reply:
Suggestion: we can also create two biweekly recurring meetings with a link to sign-up form and invite everyone. People can look at the sign-up form to see if there's a topic they are interested in. @asdine thoughts?

Reply:
Sounds good 👍